mmrotate/core/bbox/coder/delta_xywht_rbbox_coder.py

```diff
@@ -1,26 +1,26 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-import mmcv
 import numpy as np
 import torch
 from mmdet.models.task_modules.coders.base_bbox_coder import BaseBBoxCoder
 
+from mmrotate.core.bbox.structures import RotatedBoxes
 from mmrotate.registry import TASK_UTILS
 from ..transforms import norm_angle
 
 
 @TASK_UTILS.register_module()
-class DeltaXYWHAOBBoxCoder(BaseBBoxCoder):
-    """Delta XYWHA OBBox coder. This coder is used for rotated objects
+class DeltaXYWHTRBBoxCoder(BaseBBoxCoder):
+    """Delta XYWHT RBBox coder. This coder is used for rotated objects
     detection (for example on task1 of DOTA dataset). this coder encodes bbox
-    (xc, yc, w, h, a) into delta (dx, dy, dw, dh, da) and decodes delta (dx,
-    dy, dw, dh, da) back to original bbox (xc, yc, w, h, a).
+    (xc, yc, w, h, t) into delta (dx, dy, dw, dh, da) and decodes delta (dx,
+    dy, dw, dh, da) back to original bbox (xc, yc, w, h, t).
```

Comment:
Should here uses "dt" instead of "da"? May add some explanation of what "T" stands for.